config_utilities_ros/config_utilities_ros/gui/test_cfg.yaml

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
Might be missing something, but is this used anywhere?

Reply:
I thought I had cleared most of these up, not needed. Thanks for catching these!